LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2018 Mike Grip
```

Comment:
I'm perfectly fine having all 3 of us on here if you'd like!

Reply:
Alright, done! :smile: